bigquery_etl/cli/stage.py

```diff
@@ -261,14 +261,14 @@ def _update_references(artifact_files, project_id, dataset_suffix, sql_dir):
                 re.compile(
                     rf"(?<![\._])`?{original_dataset}`?\.{name}(?![a-zA-Z0-9_])`?"
```

Comment:
Seems to be missing an optional backtick before `{name}`.
```suggestion
                    rf"(?<![\._])`?{original_dataset}`?\.`?{name}(?![a-zA-Z0-9_])`?"
```